src/ansys/grpc/dpf/base_pb2.py

```diff
@@ -1,22 +1,12 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# NO CHECKED-IN PROTOBUF GENCODE
 # source: base.proto
-# Protobuf Python Version: 5.27.2
+# Protobuf Python Version: 4.25.1
 """Generated protocol buffer code."""
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import descriptor_pool as _descriptor_pool
-from google.protobuf import runtime_version as _runtime_version
```

Comment:
We can see the problematic import of `google.protobuf.runtime_version` is removed.